build-system/test-configs/forbidden-terms.js

```diff
@@ -19,6 +19,9 @@ const path = require('path');
 // Terms are defined in this file.
 /* eslint-disable local/no-forbidden-terms */
 
+// Ensures that file is up-to-date.
+/* eslint "local/forbidden-terms-config": 2 */
+
```

Comment:
For maintainability, would it make sense to move these two rules to a new file `build-system/test-configs/.eslintrc.js`?

Reply:
Sure, moved. It also happens to make linting this file faster since apparently file-level rules only omit reports, not scanning.

```console
TIMING=3 npx eslint build-system/test-configs/forbidden-terms.js
```

**Before**:
```
Rule                          | Time (ms) | Relative
:-----------------------------|----------:|--------:
prettier/prettier             |   241.615 |    64.8%
local/forbidden-terms-config   |    34.484 |     9.3%
local/no-forbidden-terms      |    30.997 |     8.3%
````

**After:**

```
Rule                          | Time (ms) | Relative
:-----------------------------|----------:|--------:
prettier/prettier             |   248.790 |    71.7%
local/forbidden-terms-config   |    34.043 |     9.8%
jsdoc/check-types             |    16.759 |     4.8%
```